src/main/scala/com/microsoft/azure/iot/kafka/connect/source/IotHubSourceTask.scala

```diff
@@ -28,6 +30,8 @@ class IotHubSourceTask extends SourceTask with LazyLogging with JsonSerializatio
       for (partitionSource <- this.partitionSources) {
         logger.debug(s"Polling for data in partition ${partitionSource.partition}")
         val sourceRecordsList = partitionSource.getRecords
+        logger.info(s"${eventHubName}:${partitionSource.partition} - Polling for data - " +
+          s"Obtained ${sourceRecordsList.length} SourceRecords")
         list ++= sourceRecordsList
```

Comment:
Can we instead reword it as - Polling for data - Obtained ${sourceRecordsList.length} source records from ${eventHubName}:${partitionSource.partition}"